flow360/component/simulation/services.py

```diff
@@ -247,6 +248,9 @@ def validate_model(
         The root item type for validation. If None then no context-aware validation is performed.
     validation_level : Literal["SurfaceMesh", "VolumeMesh", "Case", "All"] or a list of literals, optional
         The validation level, default is ALL. Also a list can be provided, eg: ["SurfaceMesh", "VolumeMesh"]
+    treat_as_file_content: bool, optional
+        If True, the behaviour of SimulationParams constructor is the same as if reading a file,
+        that includes some hash checks, version checks, and updater.
```

Comment:
```suggestion
        that includes version checks, and updater.
```
I think we do not use simulation.json's hash.

Reply:
done